docs/api/wrapper-array/setChecked.md

```diff
@@ -0,0 +1,38 @@
+## setChecked(checked)
+
+This method is an alias of the following code.
+
+```js
+wrapperArray.wrappers.forEach(wrapper => wrapper.setChecked(checked))
+```
+
+- **Arguments:**
+  - ``{Boolean} checked (default: true)``
+
```

Comment:
accidentally put two ` here.

other than that looks good... we often forget to update `WrapperArray`. Nice job.